### PR TITLE
chore: resolve audit vulnerabilities

### PR DESCRIPTION
**Pull request by bartlomiej94**

### CODE REVIEW

File: audit-ci.jsonc [hunk 1]
```diff
@@ -102,6 +107,13 @@
...
     // from: arb-bridge-peripherals>arb-bridge-eth>hardhat>semver
     // Used only in dev.
-    "GHSA-c2qf-rxjj-qqgw"
+    "GHSA-c2qf-rxjj-qqgw",
+    // https://github.com/advisories/GHSA-g4vp-m682-qqmp
+    // contracts using ERC2771Context along with a custom trusted forwarder may see
+    // _msgSender return address(0) in calls that originate from the forwarder with calldata shorter than 20 bytes
+    // from: @arbitrum/nitro-contracts>@openzeppelin/contracts-upgradeable
+    // from @arbitrum/nitro-contracts>@openzeppelin/contracts
+    //
```
Comment (bartlomiej94) [inline audit-ci.jsonc:116]: need some brief explanation why it's safe to ignore

File: audit-ci.jsonc [hunk 2]
```diff
@@ -32,6 +32,11 @@
     // Nodejs ‘undici’ Vulnerable to CRLF
     // Used only in hardhat, so only in dev. Even then we dont use remote requests.
     "GHSA-5r9g-qh6m-jxff",
+    // https://github.com/advisories/GHSA-j8xg-fqg3-53r7
+    // vulnerable to Regular Expression Denial of Service
+    // from: eslint>optionator>word-wrap
+    // word-wrap in eslint, we are on the latest major version, safe to ignore
```
Comment (spsjvc) [inline audit-ci.jsonc:38]:
```suggestion
    // https://github.com/advisories/GHSA-j8xg-fqg3-53r7
    // word-wrap vulnerable to Regular Expression Denial of Service 
    // Used only in eslint, so only in dev.
```

File: audit-ci.jsonc (hunk 1 above)
Comment (gzeoneth) [reply audit-ci.jsonc:115]: we are not affected (not using that particular contract) and also it does not affect the sdk anyway

File: audit-ci.jsonc [hunk 3]
```diff
@@ -102,6 +106,13 @@
...
     // from: arb-bridge-peripherals>arb-bridge-eth>hardhat>semver
     // Used only in dev.
-    "GHSA-c2qf-rxjj-qqgw"
+    "GHSA-c2qf-rxjj-qqgw",
+    // https://github.com/advisories/GHSA-g4vp-m682-qqmp
+    // contracts using ERC2771Context along with a custom trusted forwarder may see
+    // _msgSender return address(0) in calls that originate from the forwarder with calldata shorter than 20 bytes
+    // from: @arbitrum/nitro-contracts>@openzeppelin/contracts-upgradeable
+    // from @arbitrum/nitro-contracts>@openzeppelin/contracts
+    //
```
Comment (spsjvc) [inline audit-ci.jsonc:115]:
```suggestion
    // https://github.com/advisories/GHSA-g4vp-m682-qqmp
    // OpenZeppelin Contracts vulnerable to Improper Escaping of Output
    // from: @arbitrum/nitro-contracts>@openzeppelin/contracts-upgradeable
    // from @arbitrum/nitro-contracts>@openzeppelin/contracts
    // We don't use ERC2771Context
```

**Review by spsjvc — APPROVED** [review]

**Review by gzeoneth — APPROVED** [review]

**Pull request merged**